internal/access/resource_access_workflow.go

```diff
@@ -281,6 +307,25 @@ func (r *AccessWorkflowResource) Update(ctx context.Context, req resource.Update
 		data.ActivationExpiry = types.Int64Value(res.Msg.Workflow.ActivationExpiry.Seconds)
 
 	}
+	var defaultDuration time.Duration
+	if !data.DefaultDuration.IsNull() {
+		defaultDuration = time.Second * time.Duration(data.DefaultDuration.ValueInt64())
+
+		if defaultDuration >= accessDuration {
```

Comment:
I'm thinking that `defaultDuration == accessDuration` should be allowed. While it is technically an anti pattern I don't think it would be that farfetched that users might attempt to set default duration and max duration to the same value.
```suggestion
		if defaultDuration > accessDuration {
```